tiapp.xml

```diff
@@ -23,8 +23,11 @@
 			</dict>
 		</plist>
 	</ios>
-	<android xmlns:android="http://schemas.android.com/apk/res/android" />
-	<modules />
+	<android xmlns:android="http://schemas.android.com/apk/res/android">
+        <manifest android:versionCode="1" android:versionName="1.1.0">
+            <application android:theme="@style/kitchensink" android:hardwareAccelerated="true" android:largeHeap="true"/>
+        </manifest>
+    </android>
 	<deployment-targets>

```

Comment:
Indentation mixed up. Please use the 4-spaces-indentation like the rest of the tiapp.xml. We can merge after that, thx!